Networking/Networking/Remote/ShippingLabelRemote.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+/// Shipping Labels Remote Endpoints.
+public final class ShippingLabelRemote: Remote {
+    /// Generates shipping label data for printing.
+    /// - Parameters:
+    ///   - siteID: Remote ID of the site that owns the shipping label.
+    ///   - shippingLabelID: Remote ID of the shipping label.
+    ///   - paperSize: Paper size string (current options are "label", "legal", and "letter").
+    ///   - completion: Closure to be executed upon completion.
+    public func printShippingLabel(siteID: Int64,
+                                   shippingLabelID: Int64,
+                                   paperSize: String,
```

Comment:
nit: what do you think about passing a custom `enum` containing the 3 options here?

Reply:
yea good idea! I originally had the enum in Yosemite layer, but it does make sense for its raw value to be a networking layer detail. updated in: https://github.com/woocommerce/woocommerce-ios/pull/3155/commits/7fd4a2596885cd00b700ab2465af51f05102663d